packages/web-react/docs/stories/examples/FileUploaderWithImagePreview.stories.tsx

```diff
@@ -54,8 +54,8 @@ export const FileUploaderWithModalImagePreview = (args) => {
     imagePreview(file.name, file);
   };
 
-  const attachmentComponent = ({ id, ...props }: SpiritFileUploaderAttachmentProps) => (
-    <FileUploaderAttachment key={id} id={id} onEdit={onEdit} {...props} />
+  const attachmentComponent = ({ id, meta, ...props }: SpiritFileUploaderAttachmentProps) => (
+    <FileUploaderAttachment key={id} id={id} meta={meta} onEdit={onEdit} {...props} />
```

Comment:
Should not break, if there are no meta data it will be undefined and it's like u didn't send anything. 
Other FileUploaders work fine and all tests passed.
Just to be sure, what do you think @literat ?